.github/workflows/playwright.yml

```
@@ -0,0 +1,108 @@
+        run: pnpm exec playwright test
+
+      - name: List snapshot images from public folder
+        id: list_images
+        run: |
+          if [ -d "public/screenshots" ]; then
+            FILES=$(ls public/screenshots/*.png | xargs -n1 basename | tr '\n' ',' | sed 's/,$//')
+            echo "FILES=$FILES" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Commit snapshot images to PR branch
+        run: |
```

actionlint (1.7.7)

"github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)